### DISCUSSION TITLE
user id

### DISCUSSION THREAD

**Discussion by hopoioo** [JustOptimize/return-ShowHiddenChannels#135] [Ideas]
is there a way in the channel preview to see the name of the users who have access to the channel, instead of their user id's?

**Comment by hopoioo** [comment]
not sure if it is to do with the relatively new display name and account name system, but for me in most cases it only shows user id

**Comment by hopoioo** [comment]
nvm it was a matter of rendering the names

**Comment by JustOptimize** [comment]
It shows the id if the user is unknown, if you have the user in the cache you will see his name.
I'm not fetching users to prevent discord from detecting the use of the plugin
